index.js

```diff
@@ -0,0 +1,132 @@
+
+const arr = [];
+let id = 2;
+let emptyField = false;
+const inputs = [];
+
+// *****************MAKE NEW INPUT FIELD*****************//
+function createNewField(id) {
+    const newField = document.createElement("div");
+    newField.setAttribute("id", id);
```

Comment:
nie dodawaj JS-em każdego atrybutu html, możesz to zrobić dużo prościej korzystając z `tempalte string` i `innerHTML` albo https://developer.mozilla.org/pl/docs/Web/API/Element/insertAdjacentHTML

```js
function createField ( id ) {
return `
 <div id=${id}>
 </div>
`;
}
document.body.innerHTML = createField(id);
```